src/app/shared/restApi/apiClient.tsx

```diff
@@ -1,11 +1,11 @@
-    import axios from 'axios';
+import axios from "axios";
```

Comment:
Is the use of double quotes (") instead of single ones (') a good practice?

Reply:
My IDE automatically changes the single quotes to double quotes